diff --git a/build.rs b/build.rs
index 5be154f..7fe53aa 100644
--- a/build.rs
+++ b/build.rs
@@ -119,6 +119,7 @@ fn build(sdk_path: Option<&str>, target: &str) {
     let mut builder = bindgen::Builder::default();
 
     builder = builder.size_t_is_usize(true);
+    builder = builder.clang_arg(format!("--target={target}"));
 
     if let Some(sdk_path) = sdk_path {
         builder = builder.clang_args(&["-isysroot", sdk_path]);
@@ -136,7 +137,9 @@ fn build(sdk_path: Option<&str>, target: &str) {
     builder = builder.layout_tests(false);
 
     let meta_header: Vec<_> = headers
-        .iter()
+        ["stdint.h", "sys/types.h"]
+        .into_iter()
+        .chain(headers.iter().copied())
         .map(|h| format!("#include <{}>\n", h))
         .collect();
 
