diff --git a/include/complex.h b/include/complex.h
index 8a1f926..e0b17be 100644
--- a/include/complex.h
+++ b/include/complex.h
@@ -7,9 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_COMPLEX_H
-#define _LIBCPP_COMPLEX_H
-
+#ifndef _LIBCPP_CR_COMPLEX_H
+#define _LIBCPP_CR_COMPLEX_H
 /*
     complex.h synopsis
 
@@ -33,4 +32,4 @@
 #  endif
 #endif // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_COMPLEX_H
+#endif // _LIBCPP_CR_COMPLEX_H
\ No newline at end of file
diff --git a/include/ctype.h b/include/ctype.h
index 066f450..6f35947 100644
--- a/include/ctype.h
+++ b/include/ctype.h
@@ -7,9 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_CTYPE_H
-#define _LIBCPP_CTYPE_H
-
+#ifndef _LIBCPP_CR_CTYPE_H
+#define _LIBCPP_CR_CTYPE_H
 /*
     ctype.h synopsis
 
@@ -62,4 +61,4 @@ int toupper(int c);
 #  endif
 #endif // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_CTYPE_H
+#endif // _LIBCPP_CR_CTYPE_H
\ No newline at end of file
diff --git a/include/errno.h b/include/errno.h
index 692f00f..e1eba05 100644
--- a/include/errno.h
+++ b/include/errno.h
@@ -7,9 +7,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_ERRNO_H
-#define _LIBCPP_ERRNO_H
-
 /*
     errno.h synopsis
 
@@ -35,6 +32,9 @@ Macros:
 #    include_next <errno.h>
 #  endif
 
+#ifndef _LIBCPP_ERRNO_H
+#define _LIBCPP_ERRNO_H
+
 #  ifdef __cplusplus
 
 #    if !defined(EOWNERDEAD) || !defined(ENOTRECOVERABLE)
@@ -398,6 +398,5 @@ static const int __elast2 = 105;
 #    endif
 
 #  endif // __cplusplus
-#endif   // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
-
 #endif // _LIBCPP_ERRNO_H
+#endif   // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
diff --git a/include/fenv.h b/include/fenv.h
index 157c24f..7e9455d 100644
--- a/include/fenv.h
+++ b/include/fenv.h
@@ -7,9 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_FENV_H
-#define _LIBCPP_FENV_H
-
+#ifndef _LIBCPP_CR_FENV_H
+#define _LIBCPP_CR_FENV_H
 /*
     fenv.h synopsis
 
@@ -115,4 +114,4 @@ extern "C++" {
 #  endif // defined(__cplusplus)
 #endif   // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_FENV_H
+#endif // _LIBCPP_CR_FENV_H
\ No newline at end of file
diff --git a/include/float.h b/include/float.h
index a6e79db..5f70f81 100644
--- a/include/float.h
+++ b/include/float.h
@@ -7,9 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_FLOAT_H
-#define _LIBCPP_FLOAT_H
-
+#ifndef _LIBCPP_CR_FLOAT_H
+#define _LIBCPP_CR_FLOAT_H
 /*
     float.h synopsis
 
@@ -96,4 +95,4 @@ Macros:
 #  endif // __cplusplus
 #endif   // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_FLOAT_H
+#endif // _LIBCPP_CR_FLOAT_H
\ No newline at end of file
diff --git a/include/inttypes.h b/include/inttypes.h
index 1869284..e3ba718 100644
--- a/include/inttypes.h
+++ b/include/inttypes.h
@@ -7,12 +7,12 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_INTTYPES_H
+#ifndef _LIBCPP_CR_INTTYPES_H
 // AIX system headers need inttypes.h to be re-enterable while _STD_TYPES_T
 // is defined until an inclusion of it without _STD_TYPES_T occurs, in which
 // case the header guard macro is defined.
 #if !defined(_AIX) || !defined(_STD_TYPES_T)
-#  define _LIBCPP_INTTYPES_H
+#define _LIBCPP_CR_INTTYPES_H
 #endif // _STD_TYPES_T
 
 /*
@@ -265,4 +265,4 @@ uintmax_t wcstoumax(const wchar_t* restrict nptr, wchar_t** restrict endptr, int
 #  endif // __cplusplus
 #endif   // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_INTTYPES_H
+#endif // _LIBCPP_CR_INTTYPES_H
\ No newline at end of file
diff --git a/include/math.h b/include/math.h
index 929bef6..4cfa7c8 100644
--- a/include/math.h
+++ b/include/math.h
@@ -7,9 +7,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_MATH_H
-#  define _LIBCPP_MATH_H
-
 /*
     math.h synopsis
 
@@ -304,6 +301,9 @@ long double    truncl(long double x);
 #      include_next <math.h>
 #    endif
 
+#ifndef _LIBCPP_MATH_H
+#  define _LIBCPP_MATH_H
+
 #    ifdef __cplusplus
 
 // We support including .h headers inside 'extern "C"' contexts, so switch
@@ -496,20 +496,5 @@ using std::__math::trunc;
 } // extern "C++"
 
 #    endif // __cplusplus
+#  endif   // _LIBCPP_MATH_H
 #  endif   // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
-
-#else // _LIBCPP_MATH_H
-
-// This include lives outside the header guard in order to support an MSVC
-// extension which allows users to do:
-//
-// #define _USE_MATH_DEFINES
-// #include <math.h>
-//
-// and receive the definitions of mathematical constants, even if <math.h>
-// has previously been included.
-#  if defined(_LIBCPP_MSVCRT) && defined(_USE_MATH_DEFINES)
-#    include_next <math.h>
-#  endif
-
-#endif // _LIBCPP_MATH_H
diff --git a/include/stdatomic.h b/include/stdatomic.h
index 2991030..2ff5581 100644
--- a/include/stdatomic.h
+++ b/include/stdatomic.h
@@ -7,9 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_STDATOMIC_H
-#define _LIBCPP_STDATOMIC_H
-
+#ifndef _LIBCPP_CR_STDATOMIC_H
+#define _LIBCPP_CR_STDATOMIC_H
 /*
     stdatomic.h synopsis
 
@@ -244,4 +243,4 @@ using std::atomic_thread_fence _LIBCPP_USING_IF_EXISTS;
 #  endif // defined(__cplusplus) && _LIBCPP_STD_VER >= 23
 #endif   // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_STDATOMIC_H
+#endif // _LIBCPP_CR_STDATOMIC_H
\ No newline at end of file
diff --git a/include/stdbool.h b/include/stdbool.h
index c34853b..5aeb68c 100644
--- a/include/stdbool.h
+++ b/include/stdbool.h
@@ -7,9 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_STDBOOL_H
-#define _LIBCPP_STDBOOL_H
-
+#ifndef _LIBCPP_CR_STDBOOL_H
+#define _LIBCPP_CR_STDBOOL_H
 /*
     stdbool.h synopsis
 
@@ -41,4 +40,4 @@ Macros:
 #  endif
 #endif // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_STDBOOL_H
+#endif // _LIBCPP_CR_STDBOOL_H
\ No newline at end of file
diff --git a/include/string.h b/include/string.h
index 6bdcd6a..652c3f4 100644
--- a/include/string.h
+++ b/include/string.h
@@ -7,9 +7,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_STRING_H
-#define _LIBCPP_STRING_H
-
 /*
     string.h synopsis
 
@@ -64,6 +61,9 @@ size_t strlen(const char* s);
 #    include_next <string.h>
 #  endif
 
+#ifndef _LIBCPP_STRING_H
+#define _LIBCPP_STRING_H
+
 // MSVCRT, GNU libc and its derivates may already have the correct prototype in
 // <string.h>. This macro can be defined by users if their C library provides
 // the right signature.
@@ -110,6 +110,5 @@ inline _LIBCPP_HIDE_FROM_ABI _LIBCPP_PREFERRED_OVERLOAD char* strstr(char* __s1,
 }
 } // extern "C++"
 #  endif
-#endif // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
-
 #endif // _LIBCPP_STRING_H
+#endif // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
diff --git a/include/tgmath.h b/include/tgmath.h
index 3f8f14f..5c60eaf 100644
--- a/include/tgmath.h
+++ b/include/tgmath.h
@@ -7,9 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_TGMATH_H
-#define _LIBCPP_TGMATH_H
-
+#ifndef _LIBCPP_CR_TGMATH_H
+#define _LIBCPP_CR_TGMATH_H
 /*
     tgmath.h synopsis
 
@@ -36,4 +35,4 @@
 #  endif
 #endif // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_TGMATH_H
+#endif // _LIBCPP_CR_TGMATH_H
\ No newline at end of file
diff --git a/include/uchar.h b/include/uchar.h
index 01e1d57..ee09f93 100644
--- a/include/uchar.h
+++ b/include/uchar.h
@@ -7,9 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_UCHAR_H
-#define _LIBCPP_UCHAR_H
-
+#ifndef _LIBCPP_CR_UCHAR_H
+#define _LIBCPP_CR_UCHAR_H
 /*
     uchar.h synopsis // since C++11
 
@@ -57,4 +56,4 @@ size_t c32rtomb(char* s, char32_t c32, mbstate_t* ps);
 #  endif // _LIBCPP_CXX03_LANG
 #endif   // __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_UCHAR_H
+#endif // _LIBCPP_CR_UCHAR_H
\ No newline at end of file
diff --git a/include/wctype.h b/include/wctype.h
index 9a5b163..72fdfd5 100644
--- a/include/wctype.h
+++ b/include/wctype.h
@@ -7,9 +7,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP_WCTYPE_H
-#define _LIBCPP_WCTYPE_H
-
+#ifndef _LIBCPP_CR_WCTYPE_H
+#define _LIBCPP_CR_WCTYPE_H
 /*
     wctype.h synopsis
 
@@ -91,4 +90,4 @@ wctrans_t wctrans(const char* property);
 #  endif // __cplusplus
 #endif   // defined(__cplusplus) && __cplusplus < 201103L && defined(_LIBCPP_USE_FROZEN_CXX03_HEADERS)
 
-#endif // _LIBCPP_WCTYPE_H
+#endif // _LIBCPP_CR_WCTYPE_H
\ No newline at end of file
